Makefile

```diff
@@ -113,12 +113,22 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	make -C conmon clean
 	make -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+
+local-cross: bin/crio.cross.windows.amd64 bin/crio.cross.darwin.amd64 bin/crio.cross.linux.amd64
+
+bin/crio.cross.%:
+	@echo "==> building $@"; \
+	GOOS=`echo "$*" | cut -d . -f 1` \
+	GOARCH=`echo "$*" | cut -d . -f 2` \
+	$(GO) build -i $(LDFLAGS) -tags "containers_image_openpgp" -o bin/crio.cross."$*" $(PROJECT)/cmd/crio ||:
```

Comment:
I'd rather drop the `||:`.  If this fails, it fails.  Folks who expect failures and want to build each target anyway can use `make --keep-going …`.